README.md

```diff
@@ -1,12 +1,4 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Korron's portfolio and SPS projects.
 
```

Comment:
You should add the link to your portfolio in the README

- http://korron-shivers-portfolio.appspot.com/